cmd/kubernetes-discovery/pkg/apiserver/handler_apis_test.go

```diff
@@ -111,6 +113,10 @@ func TestAPIs(t *testing.T) {
 				{
 					ObjectMeta: api.ObjectMeta{Name: "v1.foo"},
 					Spec: apiregistration.APIServiceSpec{
+						Service: apiregistration.ServiceReference{
```

Comment:
I'm hardcoding all these for now, because we're moving towards setting them up in a status bit with a controller.